packages/rocketchat-livechat/.app/client/stylesheets/main.less

```diff
@@ -804,6 +804,10 @@ input:focus {
 	border-top-left-radius: inherit;
 }
 
+.swal2-container {
+	z-index: 10000 !important;
```

Comment:
instead of changing the third party CSS I would rather change our own..

the largest `z-index` we [have](https://github.com/RocketChat/Rocket.Chat/blob/bd15d705574320b95189844aa4dadbf76291ee00/packages/rocketchat-livechat/.app/client/stylesheets/main.less#L596) is `200`. so it would be better changing [modal's z-index](https://github.com/RocketChat/Rocket.Chat/blob/bd15d705574320b95189844aa4dadbf76291ee00/packages/rocketchat-livechat/.app/client/stylesheets/main.less#L712) instead of sweetalert's